pom.xml

```diff
@@ -31,9 +30,41 @@
     <nsxpolicysdk.version>3.1.5.0.0</nsxpolicysdk.version>
     <nsxvmcawsint.version>3.1.5.0.0</nsxvmcawsint.version>
     <nsxvmcsdkcommon.version>3.1.5.0.0</nsxvmcsdkcommon.version>
-    <vmc.draas.version>1.24.0</vmc.draas.version>
+    <vmc.draas.version>1.25.0</vmc.draas.version>
     <oidc-oauth2.version>0.0.1</oidc-oauth2.version>
+    <jaxws-ri.version>2.3.5</jaxws-ri.version>
+    <jaxb-ri.version>2.3.3</jaxb-ri.version>
   </properties>
+
+  <profiles>
+    <profile>
+      <id>jdk10</id>
+      <activation>
+        <jdk>(,10]</jdk>
+      </activation>
+    </profile>
```

Comment:
I think we don't need this empty profile. If the other doesn't activate, we just have no active profiles, which is fine.